src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.ini

```diff
@@ -383,3 +383,24 @@ COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE="Delete training record
 COM_TJCERTIFICATE_PERMISSION_DELETE_EXTERNAL_CERTIFICATE_DESC="Allow users to delete training records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE="Delete own records"
 COM_TJCERTIFICATE_PERMISSION_DELETEOWN_EXTERNAL_CERTIFICATE_DESC="Allow users to delete own training records"
+
+;Bulk Record Config
+COM_TJCERTIFICATE_TJQUEUE_SETTING="Use TjQueue while add bulk records"
+COM_TJCERTIFICATE_TJQUEUE_SETTING_DESC="Set to 'Yes' if you want to use TjQueue while add bulk records (Note: To work this functionality you need com_tjqueue and certificate plugin enable)"
+COM_TJCERTIFICATE_RECORDS_USER_SELECT_LIMIT="User limit for add bulk records"
+COM_TJCERTIFICATE_RECORDS_USER_SELECT_LIMIT_DESC="This limit used only when you are not using TjQueue and limit above 40 will result timeout."
+COM_TJCERTIFICATE_USER_LIMIT_MESSAGE="You can not add records more than %s user"
+COM_TJCERTIFICATE_BULK_RECORD_SETTINGS="Bulk Training Record"
+COM_TJCERTIFICATE_BULK_RECORD_SETTINGS_NOTE="Please note: <br> <br> <label>You need to follow below steps to make it work properly with <i>TjQueue</i>: <br><br> 1. TjQueue extension must be installed in your site <br> 2. Enable <i>TjQueue plugin for TjCertificate</i> <br> 3. Find a information below to setup a cron <br> <a href="https://github.com/techjoomla/com_tjqueue/wiki/Tj-Queue-Integration#cron-setup" target="_blank">Cron Setup
+</a><br><br> Example: <br> */1 * * * * /usr/bin/php7.3 JOOMLA_SITE/cli/tjqueue.php -t tjqueue -n 5
+</label>"
```

Comment:
There is a syntax error here. Use a single quote for the anchor tag link

Reply:
Updated